README.md

```diff
@@ -15,12 +15,22 @@ If you are looking for a version built with angular, then you should checkout th
 
 ## Quick start
 
+
 ```
-npm install 
 
```

Comment:
For the quick start guide, we can show how to clone the project and access it using cmd `cd my-project`, and we don't need to show how to install node-sass because its included in `package.json`, so just running `npm install` will add it automatically, i guess we can node js as a prerequisite tool